plugin/storage/grpc/factory_test.go

```diff
@@ -88,6 +88,7 @@ func makeMockServices() *ClientPluginServices {
 func makeFactory(t *testing.T) *Factory {
 	f := NewFactory()
 	f.InitFromViper(viper.New(), zap.NewNop())
+	f.config.ClientConfig.Endpoint = ":0"
```

Comment:
I don't really get it - the change in OTEL is to use DialContext - is it a pro-active function that actually performs the connection? If so giving it `:0` port doesn't make sense - where would it connect?

Reply:
@yurishkuro It seems like `DialContext` is a wrapper around `NewClient` and it exists `IdleMode` which presumably performs the connection (ref: https://github.com/grpc/grpc-go/blob/master/clientconn.go#L225). Using `:0` will just use any available port.